iOS/DuckDuckGo/SettingsPrivacyProtectionDescriptionView.swift

```diff
@@ -20,17 +20,17 @@
 import SwiftUI
 import DesignResourcesKit
 
-struct PrivacyProtectionDescription {
+struct PreferencesDescription {
     let imageName: String
     let title: String
     let status: StatusIndicator
     let explanation: String
 }
 
 // Universal protection description view
-struct PrivacyProtectionDescriptionView: View {
+struct PreferencesDescriptionView: View {
 
-    let content: PrivacyProtectionDescription
+    let content: PreferencesDescription
```

Comment:
If we are renaming this type and making its usage more general I think we can move this to the `Settings/Views` group (as oppose to `Settings/Sections/PrivacyProtections`. 

Also, we should update the file header with the new name and the doc comment (`/ Universal protection description view`).